### PR TITLE
fix/extend ValueHints for all AbstractAddresses

### PR DESCRIPTION
**Pull request by jkoenig134**
## Summary by CodeRabbit

- **Refactor**
	- Improved the method of copying value hints in address-related features for enhanced performance and reliability.

### CODE REVIEW

**Comment by jkoenig134** [comment]
@coderabbitai review

File: packages/content/src/attributes/types/address/DeliveryBoxAddress.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export class DeliveryBoxAddress extends AbstractAddress implements IDeliveryBoxA
     public state?: State;
 
     public static override get valueHints(): ValueHints {
-        return ValueHints.from({
+        return super.valueHints.copyWith({
```
Comment (slavistan) [inline packages/content/src/attributes/types/address/DeliveryBoxAddress.ts:67]: May I kindly ask you to briefly explain what the purpose of these changes are?

Comment (jkoenig134) [reply packages/content/src/attributes/types/address/DeliveryBoxAddress.ts:67]: Sure.

Here we are "extending" [the valueHints from AbstractAddress](https://github.com/nmshd/runtime/blob/main/packages/content/src/attributes/types/address/AbstractAddress.ts#L22) like we do with the render hints underneath.

This call now adds propertyHints for recipient to the valueHints.

This is just a bugfix and was missed before. With StreetAddress for example [we also do this properly](https://github.com/nmshd/runtime/blob/main/packages/content/src/attributes/types/address/StreetAddress.ts#L66).

Comment (jkoenig134) [reply packages/content/src/attributes/types/address/DeliveryBoxAddress.ts:67]: I you need a deeper explanation I'd suggest to not let this block the PR and we do it afterwards.

**Review by slavistan — APPROVED** [review]

**Pull request merged**